app/controllers/api/v1/users_controller.rb

```diff
@@ -18,6 +56,10 @@ def auth_user
         authorize current_user
       end
 
+      def user
+        @user ||= User.find(params[:id])
```

Comment:
`params[:id]`? I thought we were going to use the user id from the encrypted message